zeebe/backup-stores/filesystem/src/main/java/io/camunda/zeebe/backup/filesystem/FilesystemBackupStore.java

```diff
@@ -148,24 +153,32 @@ public CompletableFuture<Void> closeAsync() {
             executor.shutdown();
             final var closed = executor.awaitTermination(1, TimeUnit.MINUTES);
             if (!closed) {
-              LOG.warn("Failed to orderly shutdown Filesystem Store Executor within one minute.");
```

Comment:
💭 I didn't think this warranted a warning, since this is most likely done on shutdown.